### PR TITLE
Add buttons to scroll to top and bottom of transcript

### LINKED ISSUE #1106: Add Up/Down arrows to transcript controls
Add Up / Down arrows to transcript controls [per user story](https://docs.google.com/document/d/15Dnu6zGixnCb7UNKRcvoIt0WB6TpItmGI2SZVcZNMS8/edit#heading=h.g3tf9pw24tw8)

### PR DESCRIPTION
**Pull request by robertknight**
The icons were taken from the interactive mock at https://hypothesis-video-transcriptions.netlify.app.

Fixes https://github.com/hypothesis/via/issues/1106

- [x] TODO: Make the tests more realistic, they currently render in such a way that there is never a scroll offset

**Preview:**

Scroll to top and bottom

**Testing:**

Click the new up and down arrow buttons in the controls bar. They should scroll the transcript to the top and bottom respectively.

### CODE REVIEW

File: via/static/scripts/video_player/components/test/Transcript-test.js [hunk 1]
```diff
@@ -76,7 +87,7 @@ describe('Transcript', () => {
           currentTime={5}
           onSelectSegment={selectSegment}
         />,
-        { attachTo: document.body }
+        { attachTo: container }
```
Comment (robertknight) [inline via/static/scripts/video_player/components/test/Transcript-test.js:90]: Rendering into a separate container element resolves an issue I encountered where tests could interfere with one another.

File: via/static/scripts/video_player/components/test/Transcript-test.js [hunk 2]
```diff
@@ -168,24 +179,89 @@ describe('Transcript', () => {
...
+    beforeEach(() => {
+      wrappers = [];
+      container.style.height = '400px';
+    });
+
+    afterEach(() => {
+      wrappers.forEach(w => w.unmount());
+    });
+
+    it('scrolls to current segment when `scrollToCurrentSegment` is called', () => {
```
Comment (robertknight) [inline via/static/scripts/video_player/components/test/Transcript-test.js:223]: The tests for the existing `scrollToCurrentSegment` method have been reworked to enable them to distinguish between scrolling to the current position vs the top or bottom.

File: via/static/scripts/video_player/components/test/Transcript-test.js [hunk 3]
```diff
@@ -168,24 +179,89 @@ describe('Transcript', () => {
...
+    let wrappers;
+
+    const longTranscript = {
+      segments: Array(20)
+        .fill(0)
+        .map((_, i) => ({
+          start: i * 10,
+          text: `Text of segment ${i}`,
+        })),
+    };
```
Comment (robertknight) [inline via/static/scripts/video_player/components/test/Transcript-test.js:192]: Use a long transcript so that our test container becomes scrollable.

**Review by acelaya — APPROVED** [review]
👍🏼

**Pull request merged**